README.md

```diff
@@ -59,7 +66,8 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
-<!-- Replace this comment with your answer -->
+weather = "sunny"; is assining a string to the variable weather
```

Comment:
Although this is all true, the question was specifically asking for the values that each expression was evaluating to.

Line 1, `var weather;`, evaluates to `undefined`.
Line 2, `weather = "sunny"`, evaluates to `"sunny"` because the `=` operator always evaluates to the value that it's assigning.
Line 3, `weather === "sunny"`, evaluates to `true`.